src/headless/converse-muc.js

```diff
@@ -941,74 +942,82 @@ converse.plugins.add('converse-muc', {
                 ])].filter(n => n);
             },
 
-            getReferenceForMention (mention, index) {
-                const nicknames = this.getAllKnownNicknames();
-                const longest_match = u.getLongestSubstring(mention, nicknames);
```

Comment:
Please remove `getLongestSubstring`, given that it's no longer being called from anywhere.

Reply:
As you see it is in red, it is removed already :)